.github/workflows/go.yml

```diff
@@ -30,4 +30,6 @@ jobs:
         run: go get -v -t -d ./...
 
       - name: Test
-        run: make test
+        run: |
```

Comment:
would you add test step for both in the CI?

Reply:
do you mean running also `make integrationtest`?

Reply:
no running a tets with elements and one with electrs

Reply:
this could be done in app level tests, like testing every operator/trade/wallet methods with esplora and with elements explorers.. sounds good?